### ISSUE TITLE
Acess violation

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3581]
**Reported by Gutonunes on 2013-10-15 20:27**
Sou brasileiro e sou voluntário no Sodalício dos Cegos e estou tendo problema de conflito com o NVDA e o programa BRfácil, quando abro o Brfácil para imprimir na impressora braille com o NVDA ativo, o Brfácil dá erro e aparece a mensagem " Acess violation at adress 6CA84532 in module 'RICHED20.dll'. Read of adress 005300CE. ".
Alguém pode ajudar. 

I am Brazilian and I volunteer in the Sodality of the Blind and am having problem of conflict with NVDA and BRfácil program, when I open Brfácil printer to print in braille with NVDA active, Brfácil gives error and the message "Access Violation at adress 6CA84532 in module 'riched20.dll.' Read of adress 005300CE. ".
Can anyone help.

**Comment by ehollig** [comment]
I think this should have been closed from [e15e8ad](https://github.com/nvaccess/nvda/commit/e15e8ad9f24f3fd737ca12dd3fd127dfad438769).

**Comment by ehollig** [comment]
Closing, as this appears that it should have been closed from [e15e8ad](https://github.com/nvaccess/nvda/commit/e15e8ad9f24f3fd737ca12dd3fd127dfad438769).
